custom-pages/universal-login/src/components/Register/Register.scss

```diff
@@ -1,13 +1,16 @@
-@import '~@nice-digital/nds-core/scss/core';
+@use "@nice-digital/nds-core/scss/colours";
+@use "@nice-digital/nds-core/scss/spacing";
+@use "@nice-digital/nds-core/scss/utils";
+@use "../../styles/variables" as *;
 
 .mainpanel {
   display: flex;
   flex-direction: column;
   margin: auto;
-  min-width: var(--col-width-mobile);
+  min-width: $col-width-mobile;
 }
 .form__error {
-  color: $nds-colour-error;
+  color: colours.$error;
```

Comment:
@johnholland-nice Assuming colours is meant to be spelt "colours" (the right way ;-) and not "colors"?

Reply:
Yeah, was careful not to mix these up. The colours one comes from the design system and the color one is standard css.